ship.config.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/no-commonjs */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = {
+  mergeStrategy: { toSameBranch: ['master'] },
+  versionUpdated: ({ version, dir }) => {
+    fs.writeFileSync(
+      path.resolve(dir, 'src', 'lib', 'version.ts'),
+      `export default '${version}';\n`
+    );
+  },
+  beforeCommitChanges: ({ exec }) => {
+    exec('yarn doctoc');
+  },
+  pullRequestReviewer: ['@algolia/instantsearch-for-websites'],
+  slack: {
```

Comment:
Is this key needed?

Reply:
Ah yes!
I need to explain about this a bit.

<img width="204" alt="image" src="https://user-images.githubusercontent.com/499898/72119023-5f960680-3396-11ea-95ff-f1ab4580ae9b.png">

I was thinking of adding a slack notification on release (should've discussed earlier, though).

If we configure SLACK_INCOMING_HOOK as an env var at CircleCI, Ship.js will send notifications by default when:

1. A release PR is prepared
2. When release starts
3. When release finishes

We already assign `@algolia/instantsearch-for-websites` as reviewer, so won't need (1).

(2) is a bit excessive.

So that code block overrides (1) and (2) with null, and we will get only (3) to a slack channel. (We still need to discuss if we need this and which channel to use).

Reply:
For this non-straightforward kind of configuration, it'd be nice adding why it's here as a comment then :)

Reply:
@francoischalifour good point. added a comment: https://github.com/algolia/instantsearch.js/pull/4282/commits/972c6baa415a61a28cf6808d516bdd942e7f9b89